src/win32.mak

```diff
@@ -125,8 +125,8 @@ profile:
 
 trace: debdmd-make
 
-unittest:
-	$(DMDMAKE) "ENABLE_DEBUG=1" "ENABLE_UNITTEST=1" "ENABLE_COVERAGE=1" $(TARGETEXE)
+unittest: $(GEN)\build.exe
+	$(RUN_BUILD) unittest
```

Comment:
This is analogous to posix.mak; the previous target built a normal DMD with enabled unittests (and building the default target afterwards was a no-op...).